app/controllers/incomeSources/add/CheckUKPropertyDetailsController.scala

```diff
@@ -133,24 +133,19 @@ class CheckUKPropertyDetailsController @Inject()(val checkUKPropertyDetails: Che
   }
 
   def handleSubmit(isAgent: Boolean)(implicit user: MtdItUser[_]): Future[Result] = {
-    val redirectErrorUrl: Call = if (isAgent) routes.IncomeSourceNotAddedController.showAgent(UkProperty)
-    else routes.IncomeSourceNotAddedController.show(UkProperty)
 
     withIncomeSourcesFS {
       getUKPropertyDetailsFromSession(sessionService)(user, ec) flatMap {
         case Right(checkUKPropertyViewModel: CheckUKPropertyViewModel) =>
+          val redirectErrorUrl: Call = if (isAgent) routes.IncomeSourceNotAddedController.showAgent(UkProperty)
+          else routes.IncomeSourceNotAddedController.show(UkProperty)
```

Comment:
can this be common 
https://github.com/hmrc/income-tax-view-change-frontend/pull/1951/files#diff-449aaef5870cf3527e77495e672c27ddce127f81045da2d98e56354f5c65285eR162

Reply:
I think so, Jordan's changes will overwrite this anyway?

Reply:
ok